### ISSUE TITLE
Change tooltip content dynamically, reacting to props changes

### ISSUE THREAD

**Issue by alfonsomunozpomer** [wwayne/react-tooltip#280]
I’ve seen this has been an issue in previous issues (#231). In my opinion, it behaves as I would expect a regular React component should.

I can submit a PR if you want:
https://github.com/alfonsomunozpomer/react-tooltip/commit/14e55c6e8d8e3ca01be5dd1c8d5de0fbf5728c1b

**Comment by huumanoid** [comment]
I agree, the tooltip should respect at least changes of the `props.children`.
It would be great if you make a PR which fixes this issue!

**Comment by alfonsomunozpomer** [comment]
Done!
You may have a look here: https://github.com/wwayne/react-tooltip/pull/281
